src/RequestClientWrapper.ts

```diff
@@ -327,4 +335,33 @@ export default class RequestClientWrapper {
       return stringValue.startsWith(number.toString());
     }
   }
+
+  private isItJwtToken(token: string): boolean {
+    try {
+      JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString());
```

Comment:
What if the token is expired? I guess refreshing it will still work rigth since we have the refresh token

Reply:
@ryankazokas it won't matter. I'm just checking to see if what was entered into the session_id for the client was a JWT token so I can branch off when making a request.

Reply:
ah okay. I think it's always better to fail as soon as you know something isn't going to work. Ultimately it will throw the exception when the request fails but i think it would be better to catch, log and error here but i'm not super strong on this opinion. It will work as you have it